src/support/ssz.rs

```diff
@@ -32,15 +32,7 @@ impl<const BITS: usize, const LIMBS: usize> Decode for Uint<BITS, LIMBS> {
     }
 
     fn from_ssz_bytes(bytes: &[u8]) -> Result<Self, DecodeError> {
-        if bytes.len() > nbytes(BITS) {
-            return Err(DecodeError::InvalidByteLength {
-                len:      bytes.len(),
-                expected: nbytes(BITS),
-            });
-        }
-        let mut buf = [0u8; nbytes(BITS)]; // generic parameters may not be used in const operations
-        buf[..bytes.len()].copy_from_slice(bytes);
-        Ok(Self::from_le_bytes(buf))
+        Ok(Self::from_le_slice(bytes))
```

Comment:
still want the length check with custom error, as `from_le_slice` will panic on overflows
```
        if bytes.len() > nbytes(BITS) {
            return Err(DecodeError::InvalidByteLength {
                len:      bytes.len(),
                expected: nbytes(BITS),
            });
        }```